src/transformers/keras_callbacks.py

```diff
@@ -146,7 +146,7 @@ def _concatenate_batches(batches, padding_index=-100):
         if batches[0].ndim == 1 or all(batch.shape[1] == batches[0].shape[1] for batch in batches):
             return np.concatenate(batches, axis=0)
 
-        # Welp, they're not the same length. Let's do some padding
+        # Whelp, they're not the same length. Let's do some padding
```

Comment:
```suggestion
        # Welp, they're not the same length. Let's do some padding
```

I wrote this and it's [a real word](https://dictionary.cambridge.org/dictionary/english/welp), I promise.